assets/queries/k8s/liveness_probe_not_defined/query.rego

```diff
@@ -0,0 +1,22 @@
+package Cx
+
+CxPolicy [ result ] {
+   spec := input.document[i].spec
```

Comment:
Add metadata information to improve searchKey
document := input.document[i]
spec := document.spec
metadata := document.spec